docs/source/type_info.rst

```diff
@@ -28,6 +28,9 @@ max         float   The largest representable number.
 tiny        float   The smallest positive representable number.
 =========   =====   ========================================
 
+.. note::
+  The constructor of :class:`torch.finfo` can be called without argument, in which case the class is created for the pytorch default type (as returned by ``torch.get_default_dtype()``).   
```

Comment:
nit: 'default dtype' instead of 'default_type'

Reply:
Done